composer.json

```diff
@@ -13,12 +13,16 @@
 	"require"    : {
 		"squizlabs/php_codesniffer": "^2.9.0"
 	},
+	"suggest" : {
+		"dealerdirect/phpcodesniffer-composer-installer": "*"
```

Comment:
Wouldn't that create the risk of people blindly installing everything that's suggested and then having two plugins doing the same thing (and possibly conflicting with each other) ?

That was the reason for me to just suggest one anyway.